### PR TITLE
Contact form and final mobile version

### PR DESCRIPTION
**Pull request by MarkoKermi**
Today we were working on "Contact form" part of Portfolio project

-  Adding an HTML  for Contact form part.
- Adding an CSS for Contact form part.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -12,6 +12,29 @@ body {
   height: auto;
 }
 
+form {
+  width: 100%;
+}
```
Comment (OyePriscilla) [inline style.css]: - [ ] Please kindly ensure to layout the input elements, You can as well go through the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @MarkoKermi   😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (MarkoKermi) [reply style.css]: Dear @OyePriscilla,
Can you please give me additional explanation for your review? I'm not sure what i should change in "form" area.
Thx in advance,

Comment (OyePriscilla) [reply style.css]: @MarkoKermi Thanks for the question, though your form is properly layout but it was written according to the requirement that you should layout your form with flexbox property. Sorry for that omission! 🙏

Comment (MarkoKermi) [reply style.css]: Dear @OyePriscilla  Thx for explanation,  I see that i can remove form layout and that will not affect on my code. So is it OK like that? 
Thx in advance,

**Review by levy002 — APPROVED** [review]
## Hello @MarkoKermi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure the other parts of the website still looks as the provided design.

    - ### YOUR DESIGN
   image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (OyePriscilla) [reply style.css]: @MarkoKermi 😃

**Pull request merged**